<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VocabPage</class>
 <widget class="QWidget" name="VocabPage">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="rb_all">
        <property name="text">
         <string>全部</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_masteredOnly">
        <property name="text">
         <string>已掌握的</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="rb_word">
        <property name="text">
         <string>单词</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_sentences">
        <property name="text">
         <string>会话</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QPushButton" name="pb_deleteSelect">
     <property name="text">
      <string>删除选中内容</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_reviewMastered">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>复习已掌握内容</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
